tests/ffn_model_tests.cpp

```diff
@@ -194,3 +195,61 @@ TEST_CASE("PreTrainedResNetModel152Test", "[FFNModelsTests]")
   ResNet152 resnet152(3, 224, 224, true, true);
   PreTrainedModelTest(resnet152.GetModel(), input, 0.00199318, 0.00799561);
 }
+
+/**
+ * Simple test for MobileNetV1 model.
+ */
+TEST_CASE("MobileNetV1ModelTest", "[FFNModelsTests]")
+{
+  arma::mat input(224 * 224 * 3, 1);
+
+  // Check output shape for resnet152.
+  MobilenetV1 mobilenet(3, 224, 224);
+  ModelDimTest(mobilenet.GetModel(), input);
+}
+
+/**
+ * Test for all pre-trained MobileNetV1 models.
+ */
+TEST_CASE("PreTrainedMobileNetV1ModelTest", "[FFNModelsTests]")
+{
+  // Values taken from a PyTorch implemetation based on
+  // https://github.com/ZFTurbo/MobileNet-v1-Pytorch
+  size_t counter = 0;
+  double targets[16][3] = {{7.982727765920572e-06, 0.0008073403732851148,
+      0.0009284192346967757},
```

Comment:
Could you add a comment here, about what these values are / how we are testing this.

Reply:
Done